src/Smart.CFA.Pages/README.md

```diff
@@ -0,0 +1,3 @@
+#Client CFA
```

Comment:
You can add a space between the `#` and the text. I'm not sure what this title means btw...

Reply:
I will try to have a more self-presenting name then haha. Will do for the spacing